explainers/subresource-loading-opaque-origin-iframes.md

```diff
@@ -16,16 +16,29 @@ Support the use case of
 In this section, _the explainer_ means the [Subresource loading with Web
 Bundles] explainer.
 
-### Allow `urn:uuid:` resources
+### Allow `uuid-in-package:` resources
+
+This extension introduces a new URL scheme `uuid-in-package:` that can be used in
+resource URLs in web bundles. The `uuid-in-package:` URL has the following syntax:
+```
+<uuid-in-package> ::= "uuid-in-package:" <UUID>
+```
+Where `<UUID>` is a UUID as specified in
+[RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122).
 
 In addition to the same origin subresource explained in the
 [`<link>`-based API](https://github.com/WICG/webpackage/blob/main/explainers/subresource-loading.md#link-based-api)
 section in the explainer, this extension allows a bundle to include a
-[`urn:uuid:`](https://tools.ietf.org/html/rfc4122) URL subresource.
+`uuid-in-package:` URL subresource.
+
+Note: Previous version of this document used
```

Comment:
Nit; Can we have "Alternative Considered" section and move these into the section?

We might want to mention the reason why we no longer consider "urn:uuid", as this PR's comment explains.

Reply:
Done.